src/test/groovy/org/kohsuke/groovy/sandbox/TheTest.groovy

```diff
@@ -903,4 +903,78 @@ def cl = l.class
 return nameList.join('') + ' ' + cl
 ''')
     }
+
+    @Issue("JENKINS-65237")
+    void testGetFieldOnSuper() {
+        assertIntercept(['new A()',
+                         'A.metaClass',
+                         'HandleMetaClass.getAttribute(Class,A,String,Boolean)'],
+                "A",
+                '''
+            class A {
+              public x = 'A'
+            }
+            def a = new A()
+            def ax = a.metaClass.getAttribute(A, a, 'x', false)
```

Comment:
Since this kind of method call is never allowed in the Jenkins sandbox, it does not really tell us much. I tried testing stuff like this to see if it would trigger the GROOVY-9608 bug/fix in the unsandboxed evaluation, but it did not:

```
class A {
    def x = 1
}
class B extends A {
    def x = 2
    def test() {
        return super.@x;
    }
}
new B().test()
```

Somewhat interestingly, the sandbox evaluates that example incorrectly for whatever reason, and returns 2 rather than 1, but that's ok in some sense, because it at least intercepts things according to the incorrect evaluation (`new B()`, `new A()`, `B.test()`, `B.@x`), so there is no funny business.